File: app/src/test/java/igrek/songbook/mock/LoggerMock.java
package igrek.songbook.mock;


import android.app.Activity;

import igrek.songbook.info.logger.LogLevel;
import igrek.songbook.info.logger.Logger;

public class LoggerMock extends Logger {
	
	@Override
	public void fatal(Activity activity, String e) {
		log(e, LogLevel.FATAL, "[FATAL ERROR] ");
	}
	
	@Override
	protected void printInfo(String msg) {
		System.out.println(msg);
	}
	
	@Override
	protected void printError(String msg) {
		System.err.println(msg);
	}
	
	@Override
	protected void printDebug(String msg) {
		System.out.println(msg);
	}
	
	@Override
	protected void printWarn(String msg) {
		System.out.println(msg);
	}
	
}

File: app/src/test/java/igrek/songbook/mock/LoggerMock.kt
package igrek.songbook.mock

import igrek.songbook.info.logger.LogLevel
import igrek.songbook.info.logger.Logger

class LoggerMock : Logger() {

    override fun fatal(t: Throwable) {
        log(t.message, LogLevel.FATAL, "[FATAL ERROR] ")
    }

    override fun printInfo(msg: String) {
        println(msg)
    }

    override fun printError(msg: String) {
        System.err.println(msg)
    }

    override fun printDebug(msg: String) {
        println(msg)
    }

    override fun printWarn(msg: String) {
        println(msg)
    }

}
